examples/dask/README.md

```diff
@@ -0,0 +1,25 @@
+Dask Examples
```

Comment:
Don't you think that it will be better to place Dask examples into `python-guide` folder: `LightGBM/examples/python-guide/dask`?

Reply:
My thinking was that that makes it harder to discover the Dask examples. Because they aren't visible if you come to the repo and click "examples".

If after considering that you still think they should be moved inside `python-guide`, I'll do that. I don't feel strongly about it.

Reply:
As Dask-package is a part of LightGBM's Python-package, I thought it will be better to preserve logical hierarchy in examples.

Reply:
> Because they aren't visible if you come to the repo and click "examples".

I think you can create empty high-level folder named "Dask" and place there README file with "see `python-guide/Dask` folder [link here]". Is it OK?

Reply:
I'm even more against having an empty folder, to be honest. I'll just move these into `python-guide/dask`.

Reply:
moved in https://github.com/microsoft/LightGBM/pull/4030/commits/bef78243c2546f00ae1b98a7475f4517b6a877bb